cmake-ide.el

```diff
@@ -4,6 +4,7 @@
 
 ;; Author:  Atila Neves <atila.neves@gmail.com>
 ;; Version: 0.6
+;; Package-Version: 20190527.907
```

Comment:
Was this added manually?

Reply:
Yes, it happened accidentally. I fix it!